src/amqp.c

```diff
@@ -96,6 +96,8 @@ const char * const QD_AMQPS_PORT_STR = "5671";
 
 const char * const QD_AMQP_DFLT_PROTO = "tcp";
 
+const int PN_AMQP_MAX_SESSION_CAPACITY_32BIT = 2147483647;
```

Comment:
Was it checked if this 2^31-1 value is actually the the limit? I commented before that prior discussion suggested it might not be though it needed verified:

> Also though, if I'm remembering the last discussion of this, the value is unsigned and so wouldnt that make the configurable limit here 2^32 -1 instead really?

Seperately, should it be PN_<foo> if its being defined by Dispatch? It may be about a Proton setting, but its still a Dispatch constant. Also nothing else here or in the include file seems to use anything except QD_

Reply:
2^31-1 is not a limit imposed by Proton or by AMQP. It is half of a 32-bit address space and the chances of it being reached are about zero.

I've renamed the constant and reduced its scope in another patch.